features/digital_summary.feature

```diff
@@ -12,3 +12,13 @@ Feature: Digital Appointment Summaries
       | requested_digital |
       | requested         |
       | did not request   |
+
+  Scenario: Email notification of summary document location
+    Given the customer requested a digital appointment summary
+    When I create their record of guidance
```

Comment:
We need to go through this project at some point and change 'record of guidance' to 'appointment summary' – they both mean the same thing.